chain/jsonrpc/src/lib.rs

```diff
@@ -664,29 +689,32 @@ impl JsonRpcHandler {
         };
         timeout(self.polling_config.polling_timeout, async {
             loop {
-                let tx_status_result = self
-                    .view_client_addr
-                    .send(TxStatus {
+                let tx_status_result = message_actor(&self.view_client_addr, TxStatus {
                         tx_hash,
                         signer_account_id: account_id.clone(),
                         fetch_receipt,
                     })
                     .await;
```

Comment:
I wonder if `client_send` is a better name. 🤔 

```suggestion
                let tx_status_result = client_send(&self.view_client_addr, TxStatus {
                        tx_hash,
                        signer_account_id: account_id.clone(),
                        fetch_receipt,
                    })
                    .await;
```

Reply:
I'd probably even introduce `.client_send` and `view_client_send`